entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+APP_VERSION=${APP_VERSION:-undefined}
```

Comment:
The fallback here could be anything, I chose to put undefined so that it is apparent the value was not supplied to the `env` when booting the container.